.buildkite/e2e/nightly-main-matrix.yaml

```diff
@@ -36,12 +45,6 @@
     E2E_PROVIDER: gke
     E2E_DEPLOY_CHAOS_JOB: true
 
-- label: ocp
-  fixed:
-    E2E_PROVIDER: ocp
-  mixed:
-    - DEPLOYER_CLIENT_VERSION: "4.18.1"
```

Comment:
I don't think this is needed since the OCP version is already set here: https://github.com/elastic/cloud-on-k8s/blob/c4f3dd6302c815dfffce66e7dfd6103f6e935fa7/hack/deployer/config/plans.yml#L93-L102 ?

Reply:
Agreed